app/src/main/java/com/github/libretube/constants/Constants.kt

```diff
@@ -47,6 +47,7 @@ const val DOWNLOAD_PROGRESS_NOTIFICATION_ID = 2
 const val DOWNLOAD_CHANNEL_ID = "download_service"
 const val BACKGROUND_CHANNEL_ID = "background_mode"
 const val PUSH_CHANNEL_ID = "notification_worker"
+const val PLAYER_CHANNEL_ID = "player"
```

Comment:
The `BACKGROUND_CHANNEL_ID` already serves the purpose of creating the player notification - Therefore you likely want to remove the new `PLAYER_NOTIFICATION_ID` and rename the `BAVKGROUND_NOTIFICATION_ID` to `PLAYER_NOTIFICATION_ID`